app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,58 @@
+class RecipesController < ApplicationController
+  def index
+    @user = User.find(params[:user_id])
+    @recipes = @user.recipes.includes(:user)
+  end
+
+  def show
+    @user = current_user
+    @recipe = Recipe.find(params[:id])
+    @foods = @recipe.foods.includes(:recipe_foods).order(created_at: :desc)
+    @data = {}
+    @foods.each do |food|
+      quantity = food.recipe_foods.first.quantity
+      cost = quantity * food.price
+      @data[food.name] = { quantity:, cost:, recipe_food_id: food.recipe_foods.first.id }
```

Comment:
- [ ] [REQUIRED] Kindly ensure that this error is fixed as I can not properly navigate through the application.
![Screenshot from 2022-09-02 23-32-36](https://user-images.githubusercontent.com/58692590/188243442-bcdf19cb-4cd7-4db8-9fa7-df14a7de51ea.png)